avm/res/app/job/main.bicep

```diff
@@ -86,8 +86,8 @@ param manualTriggerConfig manualTriggerConfigType?
 @description('Optional. The maximum number of times a replica can be retried.')
 param replicaRetryLimit int = 0
 
-@description('Optional. The name of the workload profile to use.')
-param workloadProfileName string = 'Consumption'
+@description('Optional. The name of the workload profile to use. Leave empty to use a consumption based profile.')
```

Comment:
Hey @ReneHezser, really 'empty' (aka an empty string), or should the user then not specify a value? Or does both lead to 'Consumption'?

Reply:
If I may add here (I have worked with Rene on this PR, and I have submitted the original issue): It should be empty, as in the PR.

There are two types of Container App Environments:
* Consumption only environment: if your `workloadProfileName` is empty, a "consumption only" type of environment will be created
* Workload profiles environment: if you specify a non-empty `workloadprofileName`, a "workload profiles" type of environment will be created. Even if you workloadprofile nami is "Consumption"- that would be still considered as "Workload profiles" type environment.

More [here](https://learn.microsoft.com/en-us/azure/container-apps/environment)